### PR TITLE
Update isar mission definition and test

### PR DESCRIPTION
**Pull request by Christdej**

### CODE REVIEW

**Comment by Christdej** [comment]
Waiting for #607 and #609 to avoid database conflicts

File: backend/api/Controllers/Models/IsarMissionDefinition.cs [hunk 1]
```diff
@@ -18,41 +17,68 @@ public IsarMissionDefinition(List<IsarTaskDefinition> tasks)
...
             );
+            AnalysisTypes = plannedInspection.AnalysisTypes;
+            Duration = plannedInspection.TimeInSeconds;
+            var metadata = new Dictionary<string, string>
+            {
+                { "map", mission.Map.MapName },
+                { "description", mission.Description },
+                { "estimated_duration", mission.EstimatedDuration.ToString() },
+                { "asset_code", mission.AssetCode },
+                { "name", mission.Name },
```
Comment (GodVenn) [inline backend/api/Controllers/Models/IsarMissionDefinition.cs:54]:
```suggestion
                { "mission_name", mission.Name },
```

Comment (Christdej) [reply backend/api/Controllers/Models/IsarMissionDefinition.cs:54]: Agreed!

File: backend/api/Database/Models/PlannedInspection.cs [hunk 2]
```diff
@@ -16,6 +16,8 @@ public class PlannedInspection
 
         public InspectionTypeEnum InspectionType { get; set; }
 
+        public string AnalysisTypes { get; set; }
```
[the remark this answers is not in the document]
Comment (Christdej) [reply backend/api/Database/Models/PlannedInspection.cs:19]: Agreed

**Pull request merged**